### PR TITLE
docs/account: initial docset

### PR DESCRIPTION
**Pull request by d4yt0nad0cs**

### CODE REVIEW

File: src/content/docs/usage/account.md [hunk 1]
```diff
@@ -1,6 +1,131 @@
...
+
+1. Navigate to the *Settings* page under *Account* in the sidebar.
+2. Under *Git Providers*, locate the provider you wish to disconnect.
+3. Click on the three-dot menu to the right of the row.
+4. Click `Disconnect`.
+
+## Deleting your account
+You can delete your account using the Daytona dashboard.
+
+:::danger
```
Comment (ivan-burazin) [inline src/content/docs/usage/account.md:123]: @d4yt0nad0cs we use the term Caution instead of Danger

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
@d4yt0nad0cs Can we we limit number of Headings (that show up in the right side bar), and also make them shorter and that sense easier to read?

So we would go from :
Overview
Choosing the default editor used to connect to workspaces
Configuring the default dotfiles repository for new workspaces
Sharing a common environment variable across workspaces
Editing a global environment variable
Removing a global environment variable
Adding an SSH public key to your account
Removing an SSH public key from your account
Connecting an additional Git provider to your account
Disconnecting a Git provider from your account
Deleting your account

To
Overview
Setting IDE
Adding Dotfiles
Adding Environment Variables 
Adding SSH Public Key 
Connecting an Git provider 
Deleting your Account

With this there would be needed substions to be created (add edit delete) and perhaps even a intro sentence for each heading. But would be easier to navigate for the user.

**Comment by d4yt0nad0cs** [comment]
@ivan-burazin latest commit shortens the headings and restructures slightly into sections.

**Review by ivan-burazin — APPROVED** [review]
@nkkko I would approve this just please set titles to "title case" also shorted "Choosing the default editor" to Default IDE" to be consistent

cc @d4yt0nad0cs tagging you just for future reference

**Pull request merged**